src/api/server.rkt

```diff
@@ -138,12 +180,17 @@
   (place-channel-get a))
 
 (define (is-server-up)
-  (not (sync/timeout 0 manager-dead-event)))
+  (cond
+    [manager (not (sync/timeout 0 manager-dead-event))]
+    [else #t]))
 
 (define (start-job-server job-cap)
-  (define r (make-manager job-cap))
-  (set! manager-dead-event (place-dead-evt r))
-  (set! manager r))
+  (cond
+    [(> job-cap 1)
```

Comment:
I think single-threaded mode should be separate from the concept of 1 thread. Maybe just make it a separate method or a global parameter or something?